### PR TITLE
add PAPI support to items

### LINKED ISSUE #154: Lore Placeholders
Is your feature request related to a problem? Please describe.
Placeholder isnt loading on Items lores

Describe the solution you'd like
Supporting Placeholders on on Items Lores in Lobby

Describe alternatives you've considered
I dont think there's alternatives

Additional context
Nope

### PR DESCRIPTION
**Pull request by reussy**
added support for PAPI in lobby, pre-game and spectator items (item name and item lore)
added replace method for string list in SupportPAPI

Closes #154

### CODE REVIEW

File: bedwars-plugin/src/main/java/com/andrei1058/bedwars/arena/Arena.java [hunk 1]
```diff
@@ -1657,7 +1658,10 @@ public static void sendLobbyCommandItems(Player p) {
                 ItemStack i = Misc.createItem(Material.valueOf(config.getYml().getString(ConfigPath.GENERAL_CONFIGURATION_LOBBY_ITEMS_MATERIAL.replace("%path%", item))),
                         (byte) config.getInt(ConfigPath.GENERAL_CONFIGURATION_LOBBY_ITEMS_DATA.replace("%path%", item)),
                         config.getBoolean(ConfigPath.GENERAL_CONFIGURATION_LOBBY_ITEMS_ENCHANTED.replace("%path%", item)),
-                        getMsg(p, Messages.GENERAL_CONFIGURATION_LOBBY_ITEMS_NAME.replace("%path%", item)), getList(p, Messages.GENERAL_CONFIGURATION_LOBBY_ITEMS_LORE.replace("%path%", item)),
+                        SupportPAPI.getSupportPAPI().replace(p, getMsg(p, Messages.GENERAL_CONFIGURATION_LOBBY_ITEMS_NAME.replace("%path%", item))),
+                        SupportPAPI.getSupportPAPI().replace(p, getList(p, Messages.GENERAL_CONFIGURATION_LOBBY_ITEMS_LORE.replace("%path%", item))),
+                        //getMsg(p, Messages.GENERAL_CONFIGURATION_LOBBY_ITEMS_NAME.replace("%path%", item)),
```
Comment (ajgeiss0702) [inline bedwars-plugin/src/main/java/com/andrei1058/bedwars/arena/Arena.java]: These added comments seem unnecessary and you should probably remove them. Other than that, looks good to me. (also dont forget the same comments are in other places in your pr)

**Review by ajgeiss0702 — CHANGES_REQUESTED** [review]

**Review by ajgeiss0702 — APPROVED** [review]

**Comment by andrei1058** [comment]
Is it ok to use papi async?

**Comment by reussy** [comment]
> Is it ok to use papi async?

I have never tried it and I don't think it's necessary either

**Comment by ajgeiss0702** [comment]
> Is it ok to use papi async?

Unless the placeholder does something specific to minecraft that requires it to by sync (which afaik none do) it should be fine

**Comment by andrei1058** [comment]
> > Is it ok to use papi async?
> 
> I have never tried it and I don't think it's necessary either

They are used async in your context :)

**Comment by ajgeiss0702** [comment]
I’d also like to note that I parse placeholders async in ajLeaderboards and have never had any issues with that

**Pull request merged**